src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Quest {
+    private LocalDateTime from;
```

Comment:
Can we change this to `fromDate` and `toDate` just so that the variable names could be more explicit?